init.lua

```diff
@@ -737,7 +739,14 @@ function PaperWM:cycleWindowSize(direction, cycleDirection)
         return new_size
     end
 
-    local findNextSize = cycleDirection == "ascending" and findNextSizeAscending or findNextSizeDescending
+    local findNextSize
+    if cycle_direction == Direction.ASCENDING then
+        findNextSize = findNextSizeAscending
+    elseif cycle_direction == Direction.DESCENDING then
+        findNextSize = findNextSizeDescending
+    else
+        error("cycle_direction must be either Direction.ASCENDING or Direction.DESCENDING")
```

Comment:
I've been using `self.logger.e()` to log errors and bail out of functions. There's no need to crash. We just can't perform this window action with an invalid direction provided.